src/main/webapp/signup.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!-- Meta tags -->
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="style.css">
+
+    <!-- Material Design -->
+    <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+
+    <!-- Bootstrap -->
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+    
+    <title>Sign Up | Mintern</title>
+  </head>
+  <body>
+    <!-- Navbar -->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <!-- Logo -->
+      <a class="navbar-brand" href="index.html">
+        <img src="images/mintern-logo.png" height="50" alt="Mintern logo">
+      </a>
+      <button
+          class="navbar-toggler"
+          type="button"
+          data-toggle="collapse"
+          data-target="#navbarToggler"
+          aria-controls="navbarToggler"
+          aria-expanded="false"
+          aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+    </nav>
+
+    <!-- Sign Up -->
+    <div class="content">
+      <h1>Welcome to Mintern!</h1>
+      <h2>Fill the information to complete your profile.</h2>
+      <!-- Mentor Sign Up -->
+      <h1>Mentor</h1>
+      <form method="POST" action="/mentor-signup">
+        <p>First Name</p>
+        <input name="first-name"/>
+        <p>Last Name</p>
+        <input name="last-name"/>
+        <p>Username</p>
+        <input name="username"/>
+        <p>Major</p>
+        <select name="major">
+          <option value="1">ITC</option>
+          <option value="2">INT</option>
+          <option value="3">ISD</option>
+          <option value="4">ITI</option>
+          <option value="5">IMT</option>
```

Comment:
It might be good to include what these stand for, example: "ITC (whatever it stands for)"

Reply:
These will be pulled from the database, so these could not go here forever, though. Would you like for the database to include comments or have them be the full names instead of just the initials?

Reply:
I meant that the UI that the user sees should include the full description of what it stands for. The actual value doesn't matter (can be handled in the backend by the database), but it's good UX to let users know what abbreviations stand for. If this will be handled by the database later on, then I'll leave the design choice up to you!

Reply:
Yeah, we'll leave those comments for the database!